.github/README.md

```diff
@@ -39,6 +40,81 @@ cd  # in your home directory
 git clone https://github.com/torvalds/linux.git
 ```
 
+If you are compiling the kernel for the first time, install some of the packages required:
+
+On a Debian/Ubuntu distribution:
+
+
+```
+sudo apt install -y \
+  qtbase5-dev \
+  pkg-config \
+  bison \
+  flex \
+  libelf-dev \
+  libssl-dev \
+  libcap-dev \
+  llvm
+```
+
+Enable the folliwing config options to ensure for `hawk` to work:
+
+```
+make defconfig && make kvm_guest.config && \
+  scripts/config \
+  -e BPF_SYSCALL \
+  -e BPF_LSM \
+  -e BPF_JIT \
+  -e DEBUG_INFO \
+  -e DEBUG_INFO_BTF \
+  -e FTRACE \
+  -e DYNAMIC_FTRACE \
+  -e FUNCTION_TRACER && \
+  make olddefconfig
+```
+
+If you are doing kernel debugging, you may additionally want to enable:
+
+```
+scripts/config \
+  -e DEBUG_ATOMIC_SLEEP \
+  -e KASAN \
+  -e KMEMLEAK \
+  -e PROVE_LOCKING \
+  -e SECURITYFS \
+  -e CGROUP_BPF \
```

Comment:
This one does not help with debugging, it's only useful for additional BPF programs (ex: cgroup_skb)

Reply:
Removed, thanks